doc/classes/EditorSettings.xml

```diff
@@ -69,6 +69,14 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="add_shortcut">
+			<return type="void" />
+			<param index="0" name="path" type="String" />
+			<param index="1" name="shortcut" type="Shortcut" />
+			<description>
+				Adds a [param shortcut] whose path is specified by [param path].
```

Comment:
Might be worth clarifying what "path" means here (with an example?). Especially the fact that the path may contain only single slash (so e.g. `parent/name`). If there is no slash, the shortcut will display under a category with its name (e.g. `name` will be displayed as `name/name`), while more slashes will be ignored (`parent/name/surname` will display as `parent/name`).

Another thing is that custom shortcuts with unassigned events will not be saved in editor settings, so you need to add them every time. Seems like `add_shortcut()` handles it properly if added shortcut already exists. Usage example would be useful.